yew-router/examples/minimal/Cargo.toml

```diff
@@ -7,8 +7,8 @@ edition = "2018"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-yew = { path = "../../../yew" }
-yew-router = { path = "../.." }
+yew = { path = "../../../yew", features = ["std_web", "services"], default-features = false }
+yew-router = { path = "../..", features = ["std_web", "core"], default-features = false }
```

Comment:
Can't have `std_web` feature enabled in the main workspace because features are additive.

The options are:
1. Create a new workspace for stdweb powered router examples
2. Only use web-sys in the router examples

Reply:
Ok. I'll invest some time getting the examples to work in with web_sys.

Reply:
Cool, fyi we don't have a great way to run the other web-sys examples either. It's on my radar though

Reply:
I've opted for adding readmes that instruct users to use `wasm_pack` and serve the files using the custom file servers in this sub-project.